development.ini

```diff
@@ -315,7 +315,7 @@ buildroot_overrides.expire_after = 1
 important_groups = proventesters provenpackager releng security_respons packager bodhiadmin
 
 # Groups that can push updates for any package
-admin_packager_groups = provenpackager releng security_respons
+admin_packager_groups = provenpackager releng security_response
```

Comment:
The `security_response` group currently doesn't exist https://admin.fedoraproject.org/accounts/group/view/security_respons